src/Command/RegenerateEmbeddingsCommand.php

```diff
@@ -60,12 +79,22 @@ public function __invoke(OutputInterface $output): int
                 }
 
                 $image = new Imagine()->load($imageData);
-                [$face->description, $face->embeddings] = $this->pictureService->generateDescriptionAndEmbeddings($image->get(Format::ID_PNG));
+
+                switch ($field) {
+                    case 'description':
+                        [$face->description, $face->descriptionEmbeddings] = $this->pictureService->generateDescriptionAndEmbeddings($image->get(Format::ID_PNG));
+                        break;
+                    case 'image':
+                        $face->imageEmbeddings = $this->voyageAI->generateImageEmbeddings($image->get(Format::ID_PNG));
+                        break;
+                    default:
+                        $output->writeln(sprintf('Field name must be one of "image" or "description", "%s" given.', $field));
+                }
 
                 $progressBar->advance();
+                $this->dm->flush();
```

Comment:
Not sure if it's relevant with the amount of data we handle, but as an FYI if we're flushing every document by itself we should also clear the document manager afterwards. Otherwise, each flush operation will continuously get slower as there are more and more documents to check for changes.